packages/netsuite-adapter/scripts/types_generator.py

```diff
@@ -245,7 +245,7 @@ def remove_script_id_underscore_suffix(script_id_prefix):
     if has_length_limitations:
       regex_matches = re.match("[\s\S]*value can be up to (\d*) characters long\.[\s\S]*", description)
       length_limit = regex_matches.groups()[0]
-      annotations['// [CORE_ANNOTATIONS.LENGTH_LIMIT]'] = length_limit
+      annotations['[CORE_ANNOTATIONS.RESTRICTION]'] = "createRestriction({{ max_length: {0} }})".format(length_limit)
```

Comment:
Nit: rename variable (in the line above as well).
```suggestion
      annotations['[CORE_ANNOTATIONS.RESTRICTION]'] = "createRestriction({{ max_length: {0} }})".format(max_length)
```